build/ci/formatting

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "*************************"
+echo "* Checking code styling *"
+echo "*************************"
+
+go install mvdan.cc/gofumpt@latest
```

Comment:
it was intentionally not installed previously so that you can use it from a packaged source rather go install it unconditionally on your system. ..

Reply:
What do you suggest here? Simply fail the test if the package is not installed?

The package can be installed in the ci.yaml file before the execution, so it would be available for CI.

Reply:
if it's already installed, use the installed version. otherwise fail it or skip it imho. BTW, I'm not sure we need to use a 3rd party style checker when the standard one is good enough. what qualities do we gain by adding another dependency?